gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -60,20 +65,21 @@ BDEPEND="
 "
 
 src_configure() {
+	local backends="$(usex drm 'drm' '')"
+	backends+="$(usex libinput ',libinput' '')"
+	backends+="$(usex x11-backend ',x11' '')"
+	# Get rid of leading comma
+	backends="$(echo ${backends} | sed -e 's/^,//')"
 	# xcb-util-errors is not on Gentoo Repository (and upstream seems inactive?)
 	local emesonargs=(
 		"-Dxcb-errors=disabled"
 		-Dxcb-errors=$(usex xcb-errors enabled disabled)
 		$(meson_use tinywl examples)
 		-Drenderers=$(usex vulkan 'gles2,vulkan' gles2)
 		-Dxwayland=$(usex X enabled disabled)
-		-Dbackends=drm,libinput$(usex x11-backend ',x11' '')
+		-Dbackends=${backends}
```

Comment:
does the backends variable need to be quoted?  I'd prefer it.

Reply:
Will be done in a next commit

Reply:
done